dev/Sonos-Kids-Controller-master/server.js

```diff
@@ -400,4 +400,4 @@ const tryReadFile = (filePath, retries = 3, delayMs = 1000) => {
 
 // listen (start app with 'node server.js')
 app.listen(8200);
-console.log(nowDate.toLocaleString() + ": [MuPiBox-Server] App listening on port 8200");
\ No newline at end of file
+console.log(nowDate.toLocaleString() + ": [MuPiBox-Server] Server started at http://localhost:8200");
```

Comment:
I changed this since in Codespaces you can click on links containing `localhost` to open a new tab showing the web page served by this server. So this change just simplifies opening the Box UI when working in GitHub Codespaces.